test/helper.rb

```diff
@@ -146,6 +146,15 @@ def ipv6_enabled?
   end
 end
 
+def capture_stdout
```

Comment:
Move this method to `fluent/test/helpers` because that file already have `#capture_log` method.